compiler/qsc/src/lib.rs

```diff
@@ -10,7 +10,7 @@ pub mod incremental;
 pub mod interpret;
 
 pub use qsc_frontend::compile::{
-    CompileUnit, PackageStore, SourceContents, SourceMap, SourceName, TargetProfile,
+    AstPackage, CompileUnit, PackageStore, SourceContents, SourceMap, SourceName, TargetProfile,
```

Comment:
Why is `AstPackage` becoming pub here? Is this a precursor of usage somewhere else?

Reply:
That snuck in from a different change, will pull it out!